server/http/src/service.rs

```diff
@@ -124,6 +124,7 @@ where
         Ok(Response::builder()
             .status(200)
             .header(ACCESS_CONTROL_ALLOW_ORIGIN, "*")
+            .header(CONTENT_TYPE, "text/html")
```

Comment:
That one needs to be `text/plain` since we aren't actually sending HTML but plain text.